i2c-ch341-usb.c

```diff
@@ -128,8 +128,8 @@ struct ch341_pin_config ch341_board_config[CH341_GPIO_NUM_PINS] =
     {   16, CH341_PIN_MODE_OUT , "gpio1"    , 0 }, // used as output
     {   17, CH341_PIN_MODE_OUT , "gpio2"    , 0 }, // used as output
     {   18, CH341_PIN_MODE_OUT , "gpio3"    , 0 }, // used as output
-    {   19, CH341_PIN_MODE_IN  , "gpio4"    , 1 }, // used as input with hardware IRQ
```

Comment:
I would leave the default configuration as it is, since

- it is documented as a standard configuration in the ```README``` and
- an identical number of input and output ports seems to be the most common case.

The user could change the configuration according to your example in the ```README```.